Add English readme (#1)
LGBT-CN/logo: README.en.md

[简体中文](README.md) | English

# Logo

Currently using

![Logo](./v2/logo.svg)

## Index

- `v0`: Six-colour rainbow flag

> - `v1`: With the six-colour rainbow flag as the background, the logo with the flag of the People’s Republic of China

Reviewer: ```suggestion
- `v1`: Logo with a six-coloured rainbow flag on a background with the national flag of the People's Republic of China
```

> - `v2`: The logo with the equal right symbol on the background of the six-colour rainbow flag

Reviewer: ```suggestion
- `v2`: Logo with a six-coloured rainbow flag on a background with the equal right symbol
```
